notification2-examples/hello-world-microservice/src/main/java/c8y/example/notification/microservice/Properties.java

```diff
@@ -1,7 +1,8 @@
-package c8y.example.notification.helloworld;
+package c8y.example.notification.microservice;
 
 import lombok.Data;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.stereotype.Component;
 
 @Component
```

Comment:
`webSocketLibrary` field is not needed anymore as we left TooTallNate only.

Reply:
Done.